teselagen/api/build_client.py

```diff
@@ -288,33 +281,14 @@ def get_sample(
         """
         output_sample: SampleRecord | None = None
 
-        try:
-            url: str = self.sample_url.format(str(sample_id))
-            response: ResponseDict = get(
-                url=url,
-                headers=self.headers,
-            )
-            assert response['content'] is not None  # noqa: S101
-            output_sample = cast(SampleRecord, json.loads(response['content']))
-
-        except Exception as _exc:
-            # fallback to bruteforce if an error occurs
-
-            # NOTE: Since when using a method to get a single record, the user only have control over the id parameter,
-            #       we can't use the same parameters as for the method to get many records. So, we choose to use the
-            #       default parameters for the method to get many records.
-            #       Except for the `gqlFilter` parameter, which is used for efficient querying.
-            get_records = wrapped_partial(self.get_samples, gqlFilter=json.dumps({'id': str(sample_id)}))
-
-            output_sample = get_record(
-                get_records=get_records,
-                record_id=sample_id,
-            )
-
-            if output_sample is None:
-                raise RecordNotFoundError(f'Sample {sample_id} not found.') from _exc
-
-        return output_sample
+        # try:
+        url: str = self.sample_url.format(str(sample_id))
+        response: ResponseDict = get(
+            url=url,
+            headers=self.headers,
+        )
```

Comment:
Should get_sample also come with the retry decorator now that its try/catch is removed? I mean, similarly to how get_aliquot works now.

Reply:
Yeah, maybe we should add this "retry" decorator to most GET methods. However this is not a required functionality, so I would be cautious to add it. If after a while there is no problem with the decorator, we should definitely add it to more methods.